Formula/fruit.rb

```diff
@@ -0,0 +1,77 @@
+class Fruit < Formula
+  desc "Dependency injection framework for C++"
+  homepage "https://github.com/google/fruit/wiki"
+  url "https://github.com/google/fruit/archive/v3.1.0.tar.gz"
+  sha256 "d3307e02b8c85421290fc0c748ef2f964a9313596f963ae14176f35674d7230c"
+
+  depends_on "cmake" => :build
+  depends_on "boost" => :optional
+
+  def install
+    args = std_cmake_args
+    args << "-DFRUIT_USES_BOOST=False" if build.without? "boost"
+    system "cmake", ".", *args
+    system "make", "install"
+  end
+
+  test do
+    (testpath/"test.cpp").write <<~EOS
```

Comment:
Where did this example come from? Can we fetch it using a resource block?

Reply:
Test is provided from the git repo for the library (in `tests` subdirectory). It can be fetched with resource block if necessary.

Reply:
or install it in `pkgshare`